src/Nethermind/Nethermind.Optimism/CL/P2P/OptimismCLP2P.cs

```diff
@@ -22,40 +22,44 @@
 using ILogger = Nethermind.Logging.ILogger;
 using Nethermind.Merge.Plugin.Data;
 using Nethermind.Optimism.CL;
-using Nethermind.Optimism.Rpc;
-using Nethermind.Core.Crypto;
-using Nethermind.JsonRpc;
 using Snappier;
 using Nethermind.Libp2p;
 using Nethermind.Libp2p.Protocols.PubsubPeerDiscovery;
+using Channel = System.Threading.Channels.Channel;
 
 namespace Nethermind.Optimism;
 
 public class OptimismCLP2P : IDisposable
 {
+    private const int MaxGossipSize = 10485760;
```

Comment:
Where is this constant coming from?

Reply:
Let's not change P2P part. It's just copy paste from what it was before